dashboard.json

```diff
@@ -36,6 +36,11 @@
         "title": "News",
         "icon": "radio",
         "items": [
+            {
```

Comment:
@justinvp Do we serve this up anywhere anymore?

Reply:
This file (`dashboard.json`) is no longer used and can be deleted.

Reply:
Filed #634